src/measure_theory/integral/set_to_l1.lean

```diff
@@ -326,8 +326,9 @@ calc ∥∑ x in f.range, T (f ⁻¹' {x}) x∥
 ... ≤ ∑ x in f.range, ∥T (f ⁻¹' {x})∥ * ∥x∥ :
   by { refine finset.sum_le_sum (λb hb, _), simp_rw continuous_linear_map.le_op_norm, }
 
-lemma norm_set_to_simple_func_le_sum_mul_norm (T : set α → F →L[ℝ] F') {C : ℝ}
-  (hT_norm : ∀ s, ∥T s∥ ≤ C * (μ s).to_real) (f : α →ₛ F) :
+lemma norm_set_to_simple_func_le_sum_mul_norm_of_integrable (T : set α → E →L[ℝ] F') {C : ℝ}
+  (hT_norm : ∀ s, measurable_set s → μ s ≠ ∞ → ∥T s∥ ≤ C * (μ s).to_real) (f : α →ₛ E)
```

Comment:
According to our conventions, this should be
```suggestion
  (hT_norm : ∀ s, measurable_set s → μ s < ∞ → ∥T s∥ ≤ C * (μ s).to_real) (f : α →ₛ E)
```
(because when you check `hT_norm`, you want to have the stronger version in your assumptions). Same thing everywhere else in your modifications.

Reply:
It's not the best place to open that discussion again, but I feel like that decision we made at some point to sometimes write lt, and sometimes ne, is really just a hindrance. They are completely equivalent and if we wrote always the same one we could just pass it from hypothesis to lemma without touching it. Now we have to change every lt given by lemmas into ne to pass it as argument of the next theorem.
The argument "lt is stronger than ne, so we should use one in hypotheses and the other in results" seemed to make sense at first, but more and more I see only the tedious juggling it imposes, for things which are in fine equivalent ways of excluding ∞.

Rambling aside: thanks for the review! I will change those ne into lt and merge.